README.md

```diff
@@ -0,0 +1,69 @@
+# GitHub App for Issue Resolution
```

Comment:
At the top of the README, let's add a note to say it is powered by [Aider](https://aider.chat/) and [Claude 3.5 Sonnet](https://www.anthropic.com/news/claude-3-5-sonnet).